src/export/exporter.ts

```diff
@@ -17,13 +18,27 @@ export abstract class AzureMonitorBaseExporter implements BaseExporter {
 
   protected _telemetryProcessors: TelemetryProcessor[];
 
-  constructor(options: ExporterConfig = DEFAULT_EXPORTER_CONFIG) {
+  constructor(public options: ExporterConfig = DEFAULT_EXPORTER_CONFIG) {
+    const connectionString = options.connectionString || process.env[ENV_CONNECTION_STRING];
+    const instrumentationKey = options.instrumentationKey || process.env[ENV_INSTRUMENTATION_KEY];
     this._logger = options.logger || new NoopLogger();
 
     // Instrumentation key is required
-    // @todo: parse connection strings
-    if (!options.instrumentationKey) {
-      this._logger.error('No instrumentation key was provided to the Azure Monitor Exporter');
+    if (!instrumentationKey && !connectionString) {
+      this._logger.error(
+        'No instrumentation key or connection string was provided to the Azure Monitor Exporter',
+      );
+      // @todo: figure out what state the exporter should be left in here
+    }
+
+    if (connectionString) {
+      const parsedConnectionString = ConnectionStringParser.parse(options.connectionString);
```

Comment:
Should the param "connectionString"? Looks like options.connectionString may not exist?

Reply:
fixed